package.json

```diff
@@ -4,48 +4,53 @@
   "license": "MIT",
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.mjs",
-  "types": "./dist/index.d.ts",
+  "types": "./.build/types/index.d.ts",
   "files": [
+    ".build",
     "dist"
   ],
   "exports": {
     ".": {
-      "require": "./dist/cjs/index.js",
-      "import": "./dist/esm/index.mjs",
-      "default": "./dist/index.umd.js"
+      "require": "./dist/presentation-3/cjs/index.js",
+      "import": "./dist/presentation-3/index.mjs",
+      "default": "./dist/presentation3/index.umd.js"
```

Comment:
I ran into some problems with a few bundlers preferring this default to the `import` when bundling - it's not a huge issue, since this package has no dependencies. Here are the `package.json` fixes here: https://github.com/IIIF-Commons/parser/commit/e4bc241efe5661f839caf369a70f250623ca53b2 that might be worth pulling across.